test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
> I mean going forward on master here.

I'm don't think I understand. As long as we don't use any new features of `DataArrays` there is not reason to require the latest version of `DataArrays` here, right?

> Right, we just need to apply upper bounds in METADATA.

...or just tell people to `Pkg.update()`.